sql/moz-fx-data-shared-prod/search/urlbar_clients_daily/urlbar_event_clients_daily.sql

```diff
@@ -0,0 +1,25 @@
+  -- urlbar event count
+WITH
```

Comment:
Looks like you can turn this combine the two queries into one so you don't need the CTE but feel free to keep it this way if it's more readable to you.

e.g.:
```sql
SELECT
  submission_date,
  client_id,
  STRUCT(
    ARRAY_AGG(events.key) AS key,
    ARRAY_AGG(events.value) AS value
  ) AS urlbar_event
FROM
  `moz-fx-data-bq-data-science.teon.temp_urlbar_clients_daily`
CROSS JOIN
  UNNEST(urlbar_event_counts) AS events
WHERE
  events.key LIKE 'urlbar#%#%'
GROUP BY
  submission_date,
  client_id
```